h2o-py/h2o/model/autoencoder.py

```diff
@@ -17,6 +17,26 @@ def anomaly(self, test_data, per_feature=False):
             Otherwise, return the mean square error.
 
         :returns: the reconstruction error.
+
+        :examples:
+
+        >>> from h2o.estimators.deeplearning import H2OAutoEncoderEstimator
+        >>> train = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/bigdata/laptop/mnist/train.csv.gz")
+        >>> test = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/bigdata/laptop/mnist/test.csv.gz")
+        >>> predictors = list(range(0,784))
+        >>> resp = 784
+        >>> train = train[predictors]
+        >>> test = test[predictors]
+        >>> ae_model = H2OAutoEncoderEstimator(activation="Tanh",
+        ...                                    hidden=[2],
+        ...                                    l1=1e-5,
+        ...                                    ignore_const_cols=False,
+        ...                                    epochs=1)
+        >>> ae_model.train(x=predictors,training_frame=train)
+        >>> test_rec_error = ae_model.anomaly(test)
+        >>> test_rec_error
+        >>> test_recon = ae_model.predict(test)
```

Comment:
Hmm... instead of a predict example, can you add a per_features=True example?

test_rec_error_features = ae_model.anomaly(test, per_feature=True)
test_rec_error_features